### DISCUSSION TITLE
Multimodal

### DISCUSSION THREAD

**Discussion by ICHarmony** [Mobile-Artificial-Intelligence/maid#336] [Q&A]
LLaVa is getting pretty advanced now but can we use it with Maid?

**Comment by danemadsen** [comment]
> LLaVa is getting pretty advanced now but can we use it with Maid?

Not currently, would take abit to implement and I'm currently struggling just to keep local functioning at all

**Reply by ICHarmony** [reply]
I appreciate you! 🙏

**Reply by danemadsen** [reply]
I may be able to get LLaVa working with Ollama within a respectable timeframe
